BlackNectar/BlackNectar/StoresTableViewController.swift

```diff
@@ -54,19 +62,37 @@ class StoresTableViewController: UITableViewController {
         super.didReceiveMemoryWarning()
         // Dispose of any resources that can be recreated.
     }
+    @IBAction func onFilterTapped(_ sender: Any) {
+        
+        if let revealController = self.revealViewController() {
+            revealController.revealToggle(animated: true)
+        }
+
+    }
+    
+    func didApplyFilters(_ filter: SideMenuFilterViewController, restaurants: Bool, stores: Bool, openNow: Bool, distanceInMiles: Int) {
+        
+        isRestaurant = restaurants
+        isStore = stores
+        openNowSwitch = openNow
+        distanceFilter = distanceInMiles
+        
```

Comment:
Now we just need logic in here to update the table view with the specified parameters